CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# How to contribute
+
+First of all, thanks for contributing!
+
+This document provides some basic guidelines for contributing to this repository. To propose improvements, feel free to submit a PR.
+
+## Reporting a Bug - Requesting a feature - Github Issues
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues][1].
+* If you're unable to find an open issue addressing the problem, [open a new one][2].
+  - **Fill out the issue template completely**. Label the issue properly.
+    - Add `severity/` label.
+    - Add `documentation` label if this issue is related to documentation changes.
+* If you have a feature request, it is encouraged to [contact support][3] so the request can be prioritized and properly tracked.
+* **Do not open an issue if you have a question**, instead [contact support][3].
+
+## Suggesting an enhancements - Pull Requests
+
+Have you fixed an issue? Many thanks!
+
+In order to ease/speed up our review, here are some items you can check/improve when submitting your PR:
+* **Ensure an [Issue has been created](#reporting)**.
+* Avoid changing too many things at once.
+  - Make sure that your Pull Requests only fixes one Issue at the time.
+* **Write tests** for the code you wrote.
+* Make sure that **all tests pass locally**.
+* Summarize your PR with a **meaningful title** and **fill out the pull request description template completely!**
+* Add the most suitable changelog label choosing one of the following:
+  * `changelog/Added` for new features.
+  * `changelog/Changed` for changes in existing functionality.
+  * `changelog/Deprecated` for soon-to-be removed features.
+  * `changelog/Removed` for now removed features.
+  * `changelog/Fixed` for any bug fixes.
+  * `changelog/Security` in case of vulnerabilities.
+  * `changelog/no-changelog` in case this PR should not appear in the changelog at all.
+
+See [here][4] for more details about changelogs.
+
+Your pull request must pass all CI tests. If you're seeing
+an error and don't think it's your fault, it may not be! [Join us on Slack][5] or send us an email, and together we'll
+get it sorted out.
+
+### Keep it small, focused
+
+Avoid changing too many things at once. For instance if you're fixing two different
+issues at once, it makes reviewing harder and the _time-to-release_ longer.
+
+### Pull Request title
+
+Unless the PR is marked with the proper exclusion label, the title will be used
+to automatically fill the changelog entries. For this reason the title must be
+concise but explanatory.
+
+### Commit Messages
+
+Please don't be this person: `git commit -m "Fixed stuff"`. Take a moment to
+write meaningful commit messages.
+
+The commit message should describe the reason for the change and give extra details
+that will allow someone later on to understand in 5 seconds the thing you've been
+working on for a day.
+
+### Releasing
+
+The release procedure is managed by Datadog.
```

Comment:
Just saw this. I think we should have a file to explain that we would make a release pretty much at the same pace as the agent release.